x-pack/plugins/enterprise_search/public/applications/app_search/components/engine/engine_nav.tsx

```diff
@@ -171,8 +228,33 @@ export const EngineNav: React.FC = () => {
           to={getAppSearchUrl(engineRoute + ENGINE_RELEVANCE_TUNING_PATH)}
           data-test-subj="EngineRelevanceTuningLink"
         >
-          {RELEVANCE_TUNING_TITLE}
-          {/* TODO: invalid boosts error icon */}
+          <EuiFlexGroup justifyContent="spaceBetween" gutterSize="none">
+            <EuiFlexItem>{RELEVANCE_TUNING_TITLE}</EuiFlexItem>
+            <EuiFlexItem className="appSearchNavIcons">
+              {invalidBoosts && (
+                <EuiIcon
+                  type="alert"
+                  color="warning"
+                  title={i18n.translate(
+                    'xpack.enterpriseSearch.appSearch.engine.relevanceTuning.invalidBoosts',
+                    { defaultMessage: 'Invalid boosts' }
+                  )}
+                  data-test-subj="EngineNavRelevanceTuningInvalidBoosts"
+                />
+              )}
+              {unsearchedUnconfirmedFields && (
+                <EuiIcon
```

Comment:
FYI - I don't super love the screen reader effect of these icons since it reads out (e.g.) "Relevance Tuning Invalid Boosts" all together without much pause, but I'm not 100% sure how to fix this without a lot more code. Happy to investigate later as tech debt though!